profiles/common/modules/features/nexteuropa_newsroom/theme/newsroom-block.tpl.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * @file
+ * Agenda.
+ */
+?>
+<?php if ($content): ?>
+<div class="newsroom_item_container <?php echo implode(' ', $css_classes) ?>" id="newsroom-block-<?php echo $type_url; ?>">
+  <a name="newsroom-block-<?php echo $type_url; ?>"></a>
+  
+  <h3 class="newsroom_title"><?php echo l($title, $url); ?></h3>
+
+  <?php echo $content; ?>
+
+  <div class="newsroom_more">
+    <?php $title = '<span class="more">More</span> <span class="more_type"><em class="placeholder">' . $title . '</em></span>'; ?>
```

Comment:
It was done like this, because we have to use "l" function instead of a tag. I don't want separate HTML mark up from template, because it adds additional confusion and complexity.